src/components/service/ServiceCard.tsx

```diff
@@ -72,19 +91,88 @@ const ServiceCard = ({ serviceData, isDiffMode, preState, state }: ServiceCardPr
               <code className="px-1 py-0.5 rounded text-xs font-mono bg-blue-100 dark:bg-blue-900/60 text-blue-800 dark:text-blue-200">
                 aₛ
               </code>
-              Storage
+              <span className="flex items-center gap-1">
+                <span>Storage</span>
+                {!isDiffMode ? (
+                  <span className="text-xs text-muted-foreground">({counts.storage.totalCount})</span>
+                ) : (
+                  <span className="text-xs text-muted-foreground">
+                    {counts.storage.changed > 0 || counts.storage.preCount !== counts.storage.postCount ? (
+                      <>
+                        ({counts.storage.preCount}/{counts.storage.postCount}
```

Comment:
```suggestion
                        ({counts.storage.changed}/{counts.storage.preCount}
```

Similarly with all other tabs.